test/toolshed/net_test.exs

```diff
@@ -3,33 +3,55 @@ defmodule Toolshed.NetTest do
   import ExUnit.CaptureIO
   alias Toolshed.Net
 
-  describe "tping" do
-    test "can ping an IPv4 address" do
-      assert capture_io(fn ->
-               Net.tping("127.0.0.1")
-             end) =~ "Response from 127.0.0.1 (127.0.0.1): time="
-    end
+  # CI can't send ICMP packets.
+  unless System.get_env("CI") == "true" do
+    describe "tping" do
```

Comment:
It looks like CI can't send ICMP packets (`:eacces` error) so I've excluded the new tests from CI, but they run locally.